3scale/porta
# Remove obsolete prefix key and further refactoring of old code

**What this PR does / why we need it**:

This removes some obsolete code.


**Which issue(s) this PR fixes** 

-none-

**Verification steps** 

1. Make sure the tests run OK
2. Make sure that `bundle exec rake backend:fake` generates fake data successfully and fills backend redis with data, looking kind of real.

**Special notes for your reviewer**:

Some previous removal of service prefixes, making it completely useless, was introduced here: https://github.com/3scale/porta/pull/1134
